### ISSUE TITLE
Missing carriage return in help text

### ISSUE THREAD

**Issue by bentsherman** [SystemsGenetics/ACE#85]
The help text is missing a newline like so:
```
    dump: Dump the system or user metadata of a data object to standard output
          as JSON formatted text.  inject: Inject new user metadata into a data object from a JSON formatted text
          file, overwriting any user metadata the data object currently
          contains.
```

Loving the help system by the way!

**Comment by 4ctrl-alt-del** [comment]
I can't find this, when I help the command I get this:
````
$ aceex help dump
Command: aceex dump <path> [system|user]
Dumps the system or user metadata of a data object to standard output as JSON.

path: The file path to the data object whose system or user metada is dumped.

````

**Comment by bentsherman** [comment]
Sorry forgot to clarify, it's in the main help text `aceex help`.

**Comment by 4ctrl-alt-del** [comment]
Fixed in ebebbe37a54041a73b0d2781891a11bf27092175.
